toolchain/check/import_cpp.cpp

```diff
@@ -22,28 +23,26 @@ namespace Carbon::Check {
 
 // Generates C++ file contents to #include all requested imports.
 static auto GenerateCppIncludesHeaderCode(
-    llvm::ArrayRef<std::pair<llvm::StringRef, SemIRLoc>> imports)
+    Context& context, llvm::ArrayRef<Parse::Tree::PackagingNames> imports)
     -> std::string {
   std::string code;
   llvm::raw_string_ostream code_stream(code);
-  for (const auto& [path, _] : imports) {
-    code_stream << "#include \"" << FormatEscaped(path) << "\"\n";
+  for (const Parse::Tree::PackagingNames& import : imports) {
+    code_stream << "#include \""
+                << FormatEscaped(
+                       context.string_literal_values().Get(import.library_id))
+                << "\"\n";
   }
   return code;
 }
 
-auto ImportCppFiles(
-    Context& context, llvm::StringRef importing_file_path,
-    llvm::ArrayRef<std::pair<llvm::StringRef, SemIRLoc>> imports,
-    llvm::IntrusiveRefCntPtr<llvm::vfs::FileSystem> fs) -> void {
-  size_t num_imports = imports.size();
-  if (num_imports == 0) {
-    return;
-  }
-
+static auto GenerateAst(Context& context, llvm::StringRef importing_file_path,
```

Comment:
Style recommends commenting functions. Maybe:

```suggestion
// Returns an AST for the C++ imports, or null on error.
static auto GenerateAst(Context& context, llvm::StringRef importing_file_path,
```

Reply:
Done.